source/octf/interface/internal/IIoTraceParser.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright(c) 2012-2020 Intel Corporation
+ * SPDX-License-Identifier: BSD-3-Clause-Clear
+ */
+
+#ifndef SOURCE_OCTF_INTERFACE_INTERNAL_IIOTRACEPARSER_H
+#define SOURCE_OCTF_INTERFACE_INTERNAL_IIOTRACEPARSER_H
+
+#include <memory>
+#include <octf/fs/IFileSystemViewer.h>
+#include <octf/proto/parsedTrace.pb.h>
+#include <octf/proto/trace.pb.h>
+#include <octf/trace/parser/TraceEventHandler.h>
+
+namespace octf {
+class IIoTraceParser : public TraceEventHandler<proto::trace::Event> {
```

Comment:
Maybe we can rename it to IoTraceParser, because it is not pure abstract interface.